transport-classes-io_uring/src/main/java/io/netty/channel/uring/AbstractIoUringStreamChannel.java

```diff
@@ -386,6 +386,7 @@ chunkSize, nextOpsId(), bgId
         protected void readComplete0(byte op, int res, int flags, short data, int outstanding) {
             assert readId != 0;
             readId = 0;
+            assert outstanding != -1 : "multi-shot not implemented yet";
```

Comment:
Because this PR only adds multi-shot for accept?

Reply:
Yes